testing/src/test/java/io/stargate/it/cql/JavaDriverTestBase.java

```diff
@@ -61,8 +61,9 @@ protected CqlSessionBuilder newSessionBuilder() {
     config.put(
         TypedDriverOption.LOAD_BALANCING_POLICY_CLASS,
         DcInferringLoadBalancingPolicy.class.getName());
-    config.put(TypedDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(60));
-    config.put(TypedDriverOption.CONTROL_CONNECTION_TIMEOUT, Duration.ofSeconds(30));
+    config.put(TypedDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(180));
+    config.put(TypedDriverOption.METADATA_SCHEMA_REQUEST_TIMEOUT, Duration.ofSeconds(180));
+    config.put(TypedDriverOption.CONTROL_CONNECTION_TIMEOUT, Duration.ofSeconds(180));
     config.put(TypedDriverOption.REQUEST_TRACE_INTERVAL, Duration.ofSeconds(5));
```

Comment:
you may consider increasing also:
`CONNECTION_INIT_QUERY_TIMEOUT("advanced.connection.init-query-timeout"),`

Reply:
I'll probably do another PR is case tests timeout again... do not want to lose reviews and clean CI run :)